pytorch_pfn_extras/nn/parallel/distributed.py

```diff
@@ -115,9 +115,11 @@ def _broadcast(
         get_foreach_wrapper().multi_tensor_scale(src, values, 1.0)
 
 
-def _group_by_type(values: Sequence[torch.Tensor]) -> List[List[torch.Tensor]]:
+def _group_by_type(values: Sequence[Optional[torch.Tensor]]) -> List[List[torch.Tensor]]:
```

Comment:
Updates for `torch.Tensor.grad` change: https://github.com/pytorch/pytorch/commit/7843a5e88272425cd8131b16707e4786c40b6b98